chapters/annotations.tex

```diff
@@ -1117,13 +1117,16 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 
 This section describes the annotations that are used to define properties of the graphical user interface.
 
-\begin{lstlisting}[language=grammar]
-  preferred-view-annotation:
-     annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+\begin{lstlisting}[language=modelica]
+annotation(preferredView = $layer$)
```

Comment:
Missing formatting:
```suggestion
annotation(preferredView = $\mathit{layer}$)
```

Reply:
Wouldn't it also make more sense to call it the _view_?
```suggestion
annotation(preferredView = $\mathit{view}$)
```

Reply:
Yes, view seems more logical. Even evoking the MVC-paradigm.

Reply:
However, I'll actually undo that for now. Not because it's a bad change, but because it should be done consistently and not hidden inside this PR.